pkg/core/native/native_nep5.go

```diff
@@ -76,7 +76,7 @@ func newNEP5Native(name string) *nep5TokenNative {
 	md = newMethodAndPrice(n.Transfer, 1, smartcontract.NoneFlag)
 	n.AddMethod(md, desc, false)
 
-	n.AddEvent("Transfer", desc.Parameters...)
+	n.AddEvent("transfer", desc.Parameters...)
```

Comment:
We can't do it at the moment, we're targeting preview2 and it has this (wrong) name being used, we have to keep `Transfer` for now then wait for neo-project/neo#1646 resolution. In the mean time I think we can do `if !ok || (string(op) != "transfer" && string(op) != "Transfer")` for our current version, this way we'll be compatible both with current native contracts and proper NEP-5 ones.